vnext/local-cli/generator-windows/templates/cpp/proj/MyApp.vcxproj

```diff
@@ -100,7 +93,8 @@
     <Import Project="PropertySheet.props" />
   </ImportGroup>
   <ImportGroup Label="ReactNativeWindowsPropertySheets">
-    <Import Project="$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.props" Condition="Exists('$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.props')" />
+    <Import Project="$(ReactNativeWindowsDir)\PropertySheets\external\Microsoft.ReactNative.Uwp.CppApp.props" Condition="Exists('$(ReactNativeWindowsDir)\PropertySheets\External\Microsoft.ReactNative.Uwp.CppApp.props')" />
+    <Import Project="..\packages\$(WinUIPackageProps)" Condition="'$(WinUIPackageProps)'!=''" />
```

Comment:
Same as above for other new winui props and targets in here.

Reply:
@NickGerleman  that one isn't really possible because the location of the nuget packages folder relative to the current project is only known to the project, not to the External props file.

Reply:
Ah, that makes sense. It would be nice if we could keep other projects in the repo up to date with the template in this case.